src/components/GameCard/GameCard.js

```diff
@@ -2,16 +2,7 @@ import React from 'react';
 import styles from './GameCard.module.css';
 import CustomSvg from '../../helper-components/CustomSvg';
 
-const GameCard = ({ game, handleWishList }) => {
-  const [liked, setLiked] = React.useState(false);
-
-  const handleLiked = () => {
-    liked ? setLiked(false) : setLiked(true);
-    liked
-      ? handleWishList(game.name, 'remove')
-      : handleWishList(game.name, 'add');
-  };
-
+const GameCard = ({ game, handleWishList, wishList }) => {
```

Comment:
Passing the entire `wishList` to the `GameCard` component, create tight coupling between the `wishList` and the `GameCard`,  I would just pass a `isLiked` prop here which would be a boolean and  do the `wishList.some` part outside of the component. In this case it doesn't matter so much because you are not using the `GameCard` component for anything else, but imagine you would the `GameCard` for something else not related to the `wishList` implementation.